contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -237,6 +277,18 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
         lastStrikeOverrideRound = vaultState.round;
     }
 
+    /**
+     * @notice Sets a new path for swaps
+     * @param newPath is the new path
+     */
+    function setPath(bytes memory newPath) external onlyOwner nonReentrant {
+        require(
+            UniswapRouter.checkPath(newPath, USDC, vaultParams.asset),
+            "!newPath"
```

Comment:
Error message could be improved with `newPath invalid`

Reply:
Fixed. Changed message to `Invalid swap path`.